### ISSUE TITLE
Use @ngtools/webpack

### ISSUE THREAD

**Issue by Foxandxss** [preboot/angular2-webpack#197]
This will add lazy load and aot support

**Comment by Ibrahim-Islam** [comment]
Where can I find this package?

**Comment by SeerUK** [comment]
It doesn't really work well enough to use this yet, there's a _lot_ of issues with it. Once it's ready, hopefully it'll be a drop-in replacement that you don't need to adjust any code for. Right now, you have to have written your app in a certain way, with some things not even being possible.

**Comment by ekarlso** [comment]
Any idea on adding this soon ?

**Comment by Foxandxss** [comment]
Not really, Need to address some stuff first and the package has some big issues, so not too much of a rush here.

**Comment by Ibrahim-Islam** [comment]
@Foxandxss Still no plan to add it?

**Comment by Foxandxss** [comment]
Will do very very soon. I was waiting for some issues to be fixed but now we (angular 2 documentation team) are planning AOT Integration on the webpack guide so I will research the best way and update here and there.

**Comment by vatsalpande** [comment]
Hi,

Any update on this? It will be a great add on

**Comment by Foxandxss** [comment]
No, I am sorry to say that. We are really busy at Angular and since the CLI is that nice, I get less motivated :(

**Comment by SeerUK** [comment]
Yeah, that's sensible really - given that the CLI has eject too.

**Comment by abadakhshan** [comment]
hi 
I apply changes in [aot-support](/aausting/angular2-webpack/commit/1bb2fafb5e48d3d9e4fc7616800fc1eb5d913bbb)

but when run "npm  run build-aot" i get this error:

" activeModuleNotFoundError: Module not found: Error: Can't resolve './../$$_gendir/src/app/app.module.ngfactory' in 'projectPath\src'"
